### PR TITLE
feat: first examples (DCP-1590)

### PR DESCRIPTION
**Pull request by jimmymadrigal**
First stable version of frames examples:
- getting started
- poll
- media
- carousel
- quiz (redirect)

### CODE REVIEW

File: examples/carousel/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# carousel
+
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+
+## Getting Started
+
+First, run the development server:
+
+```bash
+npm run dev
```
Comment (ncpenke) [inline examples/carousel/README.md:10]: minor: no need to change now, but might be best if we just support one (with the understanding that others work)

File: examples/carousel/next.config.mjs [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
```
Comment (ncpenke) [inline examples/carousel/next.config.mjs:2]: something for later, I wonder if we just pull out frames-nextjs into it's own package to import these common settings

File: examples/carousel/tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
+{
```
Comment (ncpenke) [inline examples/carousel/tsconfig.json:1]: probably worth moving the tsconfig to a common package as well (later)

File: examples/getting-started/tailwind.config.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import type { Config } from 'tailwindcss';
```
Comment (ncpenke) [inline examples/getting-started/tailwind.config.ts:1]: same with this

**Review by ncpenke — APPROVED** [review]
Amazing work!

File: examples/carousel/next.config.mjs (hunk 2 above)
Comment (jimmymadrigal) [reply examples/carousel/next.config.mjs:2]: > something for later, I wonder if we just pull out frames-nextjs into it's own package to import these common settings

The problem is, if we move them to a common package, we would have to publish this package for someone to only clone an example, I believe.

File: examples/carousel/tsconfig.json (hunk 3 above)
Comment (jimmymadrigal) [reply examples/carousel/tsconfig.json:1]: Same here, if we reference the global tsconfig, then we would loose the ability to clone a single example, right?

**Pull request merged**